src/integrations/gl/sdl_gl_platform.cpp

```diff
@@ -227,49 +584,65 @@ void gpu_channel_set_sampler_cmd(gpu_channel* channel, gpu_sampler* sampler, u32
     (void)index;
 }
 
-void gpu_channel_set_pipeline_cmd(gpu_channel* channel, gpu_pipeline* pipeline)
+void gpu_channel_set_pipeline_cmd(gpu_channel* channel, gpu_pipeline* pipeline_handle)
 {
-    (void)channel;
-    (void)pipeline;
+    gl_device* device = (gl_device*)channel;
+    gl_pipeline pipeline = gpu_convert_handle(pipeline_handle);
+    device->current_pipeline = pipeline;
+
+    glBindVertexArray(device->dummy_vao);
```

Comment:
In theory yes -- in practise, our prototype app code in `main.cpp` uses raw OpenGL, so I figured it would be better to bind it here just in case.